src/cookie-control-config.ts

```diff
@@ -77,6 +77,14 @@ export const cookieControlConfig: CookieControlConfig = {
 					event: "cookie.analytics.revoke",
 					cookieStorage: "none",
 				});
+
+				window.localStorage.removeItem("vwoSn");
+				for (let i = 0; i < window.localStorage.length; i++) {
```

Comment:
Maybe you could use this loop instead: https://stackoverflow.com/a/37507935/486434 which I think would simplify this whole block. I think a comment would also be useful here to let someone know these are hotjar local storage items - someone might not know what _hj is for